.github/workflows/workflow.yml

```diff
@@ -28,8 +26,11 @@ jobs:
         uses: actions/setup-node@v2
         with:
           node-version: ${{ matrix.node-version }}
+      - name: Update npm
+        run: npm install -g npm@5
+        if: "${{ matrix.node-version == '8.3.0' }}"
```

Comment:
This is done to support `npm ci` (the `npm` version that comes with `8.3.0` is missing it).